source/migrate-to-atlas-cli.txt

```diff
@@ -0,0 +1,65 @@
+.. _migrate-to-atlas-cli:
+
+============================
+Migrate to the {+atlas-cli+}
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+If you have already configured {+mcli+}, you can migrate your settings to the
+{+atlas-cli+} easily. Migrating to the {+atlas-cli+} preserves existing {+mcli+}
+profiles and saved |api| keys. If you haven't already configured 
+{+mcli+}, you can proceed to :ref:`<install-atlas-cli>`.
+
+Complete the
+following steps to migrate from the {+mcli+} to the {+atlas-cli+}:
+
+.. procedure::
+
+   .. step:: Install the {+atlas-cli+}.
+
+             :ref:`<install-atlas-cli>`. If you have an existing ``mongocli.toml`` configuration file from
+             the {+mcli+},
+             {+atlas-cli+} automatically :ref:`creates a new configuration file <config-toml-location>`,
+             ``config.toml``. The {+atlas-cli+} populates ``config.toml`` with
+             the profile content and |api| keys from ``mongocli.toml``.
+
+   .. step:: Update any automation scripts.
+
+             If you use any automation scripts that rely on ``mongocli.toml`` or
+             use {+mcli+} |service| commands, update the scripts to point to the
+             :ref:`new configuration file <config-toml-location>` and use 
+             :doc:`{+atlas-cli+} commands <atlas-cli-commands>` instead.
+
+             .. note::
+
+                {+atlas-cli+} supports both {+mcli+} environment variables and
+                :ref:`{+atlas-cli+} environment variables <atlas-cli-env-vars>`,
+                so changing existing environment variables is optional. However,
+                you can use either {+mcli+}
+                environment variables or :ref:`{+atlas-cli+} environment variables <atlas-cli-env-vars>`, 
+                but not both together. If you plan to use 
+                :ref:`{+atlas-cli+} environment variables <atlas-cli-env-vars>` 
+                in the future, you
+                should change your existing {+mcli+} environment variables.
+
+   .. step:: Choose an authentication method and connect to |service|.
+
+             You can authenticate using either ``atlas auth login`` or ``atlas
+             config init``. To learn more, see :ref:`<connect-atlas-cli>`.
+
+   .. step:: Use the new :doc:`{+atlas-cli+} commands <atlas-cli-commands>`.
```

Comment:
Nit: This reads to me like the commands are new (which they won't always be). But maybe you meant that they're new to the user. Up to you if you want to change.
```suggestion
   .. step:: Use :doc:`{+atlas-cli+} commands <atlas-cli-commands>`.
```

Reply:
My thought was that the use case for using the migration page is that the MCLI commands are what they currently use and they need to use the Atlas CLI commands, which will be new to the user. So I think I will keep